test/views/payments/OpenPaymentsViewSpec.scala

```diff
@@ -210,6 +268,80 @@ class OpenPaymentsViewSpec extends ViewBaseSpec {
       element(Selectors.secondPaymentViewReturnLink).attr("href") shouldBe "/submitted/%23002"
     }
 
+    "render the correct amount for the third payment" in {
```

Comment:
Could you migrate these tests to the charge type helper spec please, where each charge can have a seperate `when` section dedicated to it.

Reply:
Where is this helper.
I sdon't mind adding helper tests but will do so in addition as one tests the helper the other tests what we expect on the view is present